.github/workflows/automerge.yml

```diff
@@ -0,0 +1,15 @@
+name: Auto-merger
+
+on:
+  pull_request:
+
+jobs:
+  optimize-auto-merge:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.title == 'Optimize SVG(s)'
```

Comment:
Checking only for the title is not ideal

See https://github.com/LawnchairLauncher/lawnchair/blob/13-dev/.github/workflows/ci.yml#L109-L119 for a better implementation